pkg/operator/ceph/object/livenessprobe.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2020 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package object allows a ceph config file to be stored in Kubernetes and mounted as volumes into
+// Ceph daemon containers.
+package object
+
+import (
+	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+	rookv1 "github.com/rook/rook/pkg/apis/rook.io/v1"
+	v1 "k8s.io/api/core/v1"
+)
+
+type fn func(cephv1.BucketHealthCheckSpec) *v1.Probe
+
+// configureLivenessProbe returns the desired liveness probe for a given daemon
+func configureLivenessProbe(daemon rookv1.KeyType, container v1.Container, healthCheck cephv1.BucketHealthCheckSpec) v1.Container {
+	// Map of functions
+	probeFnMap := map[rookv1.KeyType]fn{
```

Comment:
You don't need that since you have a single function only.

Reply:
Done